backend/medtagger/repositories/slices.py

```diff
@@ -26,10 +26,18 @@ def get_slices_by_scan_id(scan_id: ScanID, orientation: SliceOrientation = Slice
     return slices
 
 
-def delete_slice_by_id(slice_id: SliceID) -> None:
+def delete_slice(_slice: Slice) -> None:
     """Remove Slice from SQL database and Storage."""
+    delete_slice_by_id(_slice.id, _slice.scan_id)
+
+
+def delete_slice_by_id(slice_id: SliceID, scan_id: ScanID) -> None:
```

Comment:
Do we need this function? Maybe merge its content to `delete_slice()`? This function is dangerous because it doesn't validate if Slice with `slice_id` belongs to Scan with `scan_id`.